examples/kyb-app/src/domains/collection-flow/collection-flow.api.ts

```diff
@@ -72,3 +72,7 @@ export const updateFlow = async (dto: UpdateFlowDto) => {
 export const startFlow = async (flowId: string) => {
   await request.post(`collection-flow/finish/${flowId}`);
 };
+
+export const resubmitFlow = async (flowId: string) => {
```

Comment:
is it possible to parse it using query params?
i want to pass workflowId + resubmitEvent 
and from there you receive the resubmission event and the workflow id for that.
so you won't need to enter your email once again?

Reply:
e.g. 
resubmissionLink: join('',[entity.data.additionalInfo.normalizedCustomerCompany,'.demo.ballerine.app','/workflowRuntimeId=',workflowRuntimeId,'?resubmitEvent=RESUBMITTED']),